charts/victoria-metrics-anomaly/values.yaml

```diff
@@ -174,10 +174,10 @@ monitoring:
   # /metrics server.
   pull:
     # -- if true expose metrics on /metric endpoint
-    enable: true
+    enabled: true
```

Comment:
this is a part of vmanomaly config :) we can do mapping and I prefer enabled

Reply:
added a mapping to leave `enable` to vmanomaly config